k8s/helm-chart/dynatrace-gcp-function/values.yaml

```diff
@@ -37,6 +40,10 @@ useProxy: ""
 httpProxy: ""
 # httpsProxy: set the https proxy address. To be used in conjunction with USE_PROXY.
 httpsProxy: ""
+# Import predefined dashboards for selected services. Allowed values: "true"/"yes", "false"/"no"
+importDashboards: "true"
```

Comment:
can we make boolean values consistent in this file?
I see some boolean vars are without quotes (lines 17, 25)